src/action/log.js

```diff
@@ -49,6 +51,49 @@ export function error(...args) {
   _ipc && _ipc.send('log-error', null, args);
 }
 
+/**
+ * Gets the location of the LND log file as a string.
+ * @param {string} network The network the user's on
+ * @return {string}
+ */
+export function getLogPath(network) {
+  if (!_FS) {
+    throw new Error('Cannot get log path with no FS in action/log.js');
+  }
+  const lndDir = _FS.DocumentDirectoryPath;
+  return `${lndDir}/logs/bitcoin/${network}/lnd.log`;
+}
+
+/**
+ * Retrieves the entire LND log file as a string.
+ * @return {Promise}
+ */
+export function getLogs() {
+  if (!_FS) {
+    return Promise.reject(
+      new Error('Cannot get logs with no FS in action/log.js')
+    );
```

Comment:
If we declare `async function getLogs` we be can throw an Error here directly. No need to wrap in `Promise.reject()` in that case.